django/contrib/admin/static/admin/css/base.css

```
@@ -937,7 +937,7 @@ a.deletelink:focus, a.deletelink:hover {
     display: flex;
 }
 
-#branding h1 {
+#branding #site-name {
```

Here and everywhere else – let’s simplify the selector so we don’t let specificity creep up.

```suggestion
#site-name {
```